plugins/palo_alto_pan_os/help.md

```diff
@@ -54,6 +54,10 @@ Example input:
 
 This action adds address objects to an address group. This action supports FQDNs, IPv4 and IPv6 addresses.
 
+##### Support
+
```

Comment:
This should not be in the support section. This information should be included in the action description. We cannot use "Support" tag as it is not supported by our tooling.